Header header
int32 id
string name
geometry_msgs/Vector3Stamped[] objects_in_view